packages/transaction-controller/src/utils/batch.ts

```diff
@@ -46,6 +54,12 @@ import {
   TransactionType,
 } from '../types';
 
+type UpdateBatchMetadata = (
```

Comment:
Minor, this is technically just a generic `UpdateStateCallback`?

Reply:
Done.